lib/OpenQA/Setup.pm

```diff
@@ -213,7 +213,7 @@ sub read_config ($app) {
         },
         test => {
             auth => {
-                method => 'Fake',
+                #method => 'Fake',
```

Comment:
Are these commented-out lines still required? If yes, they should maybe be removed completely - although I'm not sure what this will break.

Reply:
thank you. Fixed. I don't know how I managed to include those local testing changes again.